README.md

```diff
@@ -60,3 +62,44 @@ Then use the `analyticsClient` variable:
 ```js
 const urlParamsHandler = new analyticsClient.AnalyticsUrlParams();
 ```
+
+## Configuration
+
+We use environment configuration on the [analytics-backend](https://github.com/balena-io/analytics-backend) to mask the actual API tokens of the final destination services of these events.
```

Comment:
This section is quite verbose TBH and it includes a lot of internals the user doesn't have to worry about. How about just writing down a configuration example code, and then add comments per configuration entry (eg. `projectName - a unique name for your project. Reach out to the data team when in doubt about the name you should use`)